rcongui/package.json

```diff
@@ -22,6 +22,7 @@
     "material-ui": "^0.20.2",
     "moment": "^2.24.0",
     "mui-datatables": "^3.7.4",
+    "prop-type": "^0.0.1",
```

Comment:
Please remove this. I don't know what that dependency is but it smells fishy. And react prop types are already included as a dependency.